gradle/libs.versions.toml

```diff
@@ -104,7 +104,7 @@ hilt-gradle-plugin = { module = "com.google.dagger:hilt-android-gradle-plugin",
 
 # Google
 google-services-plugin = { module = "com.google.gms:google-services", version = "4.4.2" }
-google-playservices-auth = { module = "com.google.android.gms:play-services-auth", version = "21.2.0" }
+google-play-asset = { module = "com.google.android.play:asset-delivery", version = "2.2.2" }
```

Comment:
This library must be removed. That's caused by an incorrect fix of the merge conflict

Reply:
Because it was removed in another PR before yours that's already been merged

Reply:
https://github.com/Ivy-Apps/ivy-wallet/pull/3330